numpy/core/shape_base.py

```diff
@@ -416,6 +416,13 @@ def stack(arrays, axis=0, out=None):
     return _nx.concatenate(expanded_arrays, axis=axis, out=out)
 
 
+# Internal functions to elimainate the overhead of repeated dispatch inside
```

Comment:
spelling of eliminate. Also maybe write "repeated dispatch in one of the two possible paths inside np.block"